blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -76,14 +78,7 @@ class RunDetailsHeader extends Component {
                             </span>
                         </div>
                         : null }
-                        <div>
-                       { authors.length > 0 ?
-                                   <a className="authors" onClick={() => this.handleAuthorsClick()}>
-                                        Changes by {authors.map(
-                                        author => ` ${author}`)}
-                                   </a>
-                       : 'No changes' }
-                        </div>
+                        <ChangeSet {...changeProps} />
```

Comment:
I know you like this, but I am actually strongly opposed to this coding style, creating a property in a different location, when this is the location it's used is not good for maintenance, much prefer setting props directly. :ant: